temporalio/workflow.py

```diff
@@ -1878,6 +1883,7 @@ def start_activity(
     cancellation_type: ActivityCancellationType = ActivityCancellationType.TRY_CANCEL,
     activity_id: Optional[str] = None,
     versioning_intent: Optional[VersioningIntent] = None,
+    priority: Optional[temporalio.common.Priority] = None,
```

Comment:
Don't forget `start_activity_class`, `execute_activity_class`, `start_activity_method`, and `execute_activity_method`

Reply:
Done. There's a lot of those!